SpriterDotNet/Parser/XmlSpriterParser.cs

```diff
@@ -24,7 +26,14 @@ public Spriter Parse(string data)
 
         public bool CanParse(string data)
         {
-            return data.StartsWith(XmlStart);
+            return data != null && data.StartsWith(XmlStart);
+        }
+
+        private static string FixBadNanValue(string data)
+        {
+            var nanRegex = new Regex(@"(a)=""nan""");
+            data = nanRegex.Replace(data, @"$1=""NaN""");
```

Comment:
This is an excellent idea. It's much easier to have it in the parser and remove it later. However, the "nan" value should be replaced with 0, not NaN since that value is used for interpolation and is supposed to be in the [0, 1] interval. Using NaN in any arithmetic operation ends up badly.